data/json/items/tools.json

```diff
@@ -3414,7 +3414,15 @@
     "ammo": "NULL",
     "revert_to": "null",
     "category": "weapons",
-    "use_action": "MOLOTOV"
+    "use_action": {
+        "type": "transform",
+        "target": "molotov_lit",
+        "target_charges": 1,
+        "active": true,
+        "need_fire": 1,
+        "need_fire_msg": "You need a source of fire!",
+        "msg": "You light the Molotov cocktail."
+    }
```

Comment:
Old behavior had a move cost associated with it.
150 was a fair bit too much (currently lighting a rag on the ground takes like 25 or so), but is that removal intended?